sw/device/silicon_creator/manuf/data/manuf_testplan.hjson

```diff
@@ -0,0 +1,408 @@
+            hidden from software. These secrets are use to as the root of the key derivation
+            function in the key manager.
+
+            ## Test steps
+
+            ### Prep steps
+            - Pre-load the OTP state with an image of an individualized device. See
+              manuf_ft_sku_individualization for more details.
+            - Check expected ls_state and OTP locked partitions (SECRET1, HW_CFG).
+            - Initialize entropy source in FIPS mode. Check health status after setting appropriate
+              health test parameters.
+
+            ### Device secrets
+            - Instantiate the SW CSRNG, and configure it to generate data.
+            - Configure the following OTP entries with data extracted from the CSRNG:
+              - CREATOR_ROOT_KEY_SHARE0
+              - CREATOR_ROOT_KEY_SHARE1
+            - Configure the flash info secret partition with data extracted from the CSRNG.
+            - Lock SECRET2 OTP partition.
+            - Perform reset triggered by software.
+
+            ### RMA provisioning
+            - Instantiate the SW CSRNG, and configure it to generate data.
+            - Configure RMA_TOKEN in the OTP partition with data extracted from the CSRNG.
+            - Encrypt RMA_TOKEN with asymmetric algorithm (e.g RSA-3072). The encrypted blob will
+              be sent to the host in the result status.
+            - Uninstantiate the sw CSRNG.
+
+            ### Device Identity
+
+            The OWNER_SW_CFG_ROM_KEYMGR_ROM_EXT_MEAS_EN OTP must be configured so that the
+            attestation measurement can be provided in the manifest of the test program. This
+            way the provisioning program is able to generate the key manager derivation matching
+            the production ROM_EXT configuration.
+
+            TODO: Left details vague on purpose until we get the attestation flow reviewed.
+            - Advance the keymgr to CREATOR_ROOT_KEY state.
+            - Derive CREATOR attestation key pair.
+            - Calculate SHA2 hash of the public key with the device secret provisionined in
+              the manuf_cp_device_info_flash_wr test. The device secret is known by the
+              provisioning appliance.
+            - Disable the key manager.
+            - Configure OWNER_SW_CFG_ROM_KEYMGR_ROM_EXT_MEAS_EN OTP to switch to attestation mode
+              associated with the device SKU.
+
+            ### Result status
+            - Send response to host with test result serialized in JSON format. The result must
+              include the encrypted RMA_UNLOCK_TOKEN, the DEVICE_ID, the public attestation key
+              and its hash.
+            '''
+      tags: ["dv", "fpga", "silicon"]
+      stage: V3
+      tests: []
+    }
+
+    {
+      name: manuf_ft_load_transport_image
+      desc: '''Load transport stack into flash.
+
+            Ensure the ROM is able to bootstrap the transport image. The transport image supports
```

Comment:
I thought the transport image is the 3rd stage, so ROM_EXT, not ROM, should be able to boot it, right?